cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/LocationGroupEndpointInput.java

```diff
@@ -142,11 +155,13 @@ public static final class Patch extends EndpointInput {
 
         private final LocationGroup locationGroup;
         private final String originalGroupId;
-        private boolean replaceAssignedLocs;
+        private boolean replaceAssignedLocs = false;
+        private final String office;
 
-        private Patch(String originalGroupId, LocationGroup locationGroup) {
+        private Patch(String office, String originalGroupId, LocationGroup locationGroup) {
```

Comment:
What office is this?

Reply:
Updated to reflect that this is the user operating office

Reply:
This was part of a patch to avoid the unauthorized request error caused when trying to unassign locations from a CWMS location group. The documentation in the CDA controller identified the parameter as the operating office of the current user, but it would be possible to use the location office as long as it is not CWMS. I'll rename this to locationOffice.